src/NServiceBus.Hosting.Windows/NServiceBus.Host.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
```

Comment:
Does this require us to release the packages?

Reply:
I wouldn't think so - we are not making any changes that are of benefit to anyone?